terrautils/betydb.py

```diff
@@ -47,6 +48,128 @@ def get_bety_key():
                        "or create $HOME/.betykey.")
 
 
+def get_brapi_url(path=''):
+    """return brapi url from environment with optional path
```

Comment:
see comment below, but i'd like to see a simple brapi_get(path, params) that returns all the results. try not to have too many 1 line function calls.

Reply:
done. there is now a brapi_get

the brapi URL and version are now in the function or environment variables